Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/LayerTreeHost.cpp

```diff
@@ -366,9 +367,17 @@ void LayerTreeHost::deviceOrPageScaleFactorChanged()
     m_compositor->setScaleFactor(m_webPage.deviceScaleFactor() * m_viewportController.pageScaleFactor());
 }
 
+PlatformDisplayID LayerTreeHost::displayID() const
+{
+    return m_compositor->displayRefreshMonitor().displayID();
+}
```

Comment:
Display ID value isn't dependent on the DisplayRefreshMonitor object. To me it makes more sense to calculate it in the LayerTreeHost constructor, like proposed. and store it in a member value like, it's done today.

Reply:
It's stored by the refresh monitor, so I thought we could avoid storing twice, but it's not a big difference so I'll bring it back to LayerTreeHost